modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -396,9 +396,13 @@ State:                      ready
 Private:                    No
 Created:                    Oct  1 2021 08:12:25 UTC
 Details Page:               https://console.redhat.com/openshift/details/s/<subscription_id>
-OIDC Endpoint URL:          https://rh-oidc.s3.<aws_region>.amazonaws.com/<cluster_id>
+OIDC Endpoint URL:          https://oidc.op1.openshiftapps.com/<cluster_id>|<oidc_config_id> \ <1>
 ----
 +
+--
+1. The endpoint URL depends on the BYO OIDC configuration. If you are pre-creating the OIDC configuration, the URL should end with `<oidc_config_id>`; otherwise, it should be the `<cluster-ID>`.
```

Comment:
```suggestion
1. The endpoint URL depends on the BYO OIDC configuration. If you are pre-creating the OIDC configuration, append the `<oidc_config_id>` value to the URL; otherwise, append the URL with the `<cluster-ID>` value.
```
Per the ISG don't use "should" to mean "must": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s. Consider my suggested edit as one way to avoid using "should", and to make the action more user-centric.